docs/manual/scala/guide/cluster/DomainModelling.md

```diff
@@ -1,19 +1,121 @@
 # Domain Modelling
 
+We recommend reading [[Event Sourcing and CQRS|ES_CQRS]] as a prerequisite to this section.
+
+In this section we will guide you through all the necessary steps to model an [Aggregate](https://martinfowler.com/bliki/DDD_Aggregate.html), as defined in Domain-Driven Design, using Akka Persistence Typed and following the CQRS principles.
+
+Lagom embraces Event Sourcing and CQRS by promoting the usage of Akka Persistence Typed in combination the CQRS principle. Akka Persistence Typed provides an API for building event sourced actors, but not necessarily for CQRS Aggregates. To build CQRS applications we need to apply a few rules to our design.
+
+We will explain the API and concepts while building a simplified shopping cart model. A full-fledge shopping cart sample can be found on our [samples repository](https://github.com/lagom/lagom-samples/tree/1.6.x/shopping-cart/shopping-cart-scala).
```

Comment:
Regarding the shopping-cart example, I think it would be better to use the same domain used in Lagom's persistence API docs. That way it will be easier for users to compare both examples.

Reply:
IMHO, we should migrate the classic docs, then. We're moving everything to shopping-cart.

Reply:
My initial idea was to indeed use the BlogPost example.

My motivation to use the shopping cart, is that we can explain a few important aspects without forcing the hand. For instance, the "changing behavior" section doesn't really show how to model it because the blog post is too simple. Error handling is also not great with BlogPost, IMO.

Then I also thought that it would be a good idea to have shopping-cart everywhere. 

I would prefer not to migrate the classic docs right now. I'm ok in doing that later, but if we say that it must be a requirement to have them aligned, I would choose for the least-effort, being using BlogPost on the new docs.

Reply:
It'd be ideal to have the classic docs using shopping cart, but it may be too much work. I like @renatocaval's points wrt to shopping cart being an example that easily helps demo some features.

I've reconsidered and the price to rewrite classic docs to use shopping cart may be too high. I think the best trade-off is to keep classic using blogpost and akka typed using shopping cart.